examples/blog/keystone.ts

```diff
@@ -1,10 +1,14 @@
 import { config } from '@keystone-next/keystone/schema';
 import { lists } from './schema';
+import { extendGraphqlSchema } from './extendGraphqlSchemas'
```

Comment:
Example "blog" must be as simple as possible, so better to not change it, and implement separate example instead, or extend the [examples/extend-graphql-schema](https://github.com/keystonejs/keystone/tree/master/examples/extend-graphql-schema) example with this feature, or provide an example in this feature description as documentation.

Reply:
I create new example prisma-preview-features